workers/unity/Assets/Playground/Scripts/DisconnectSystem.cs

```diff
@@ -11,17 +11,20 @@ internal class DisconnectSystem : ComponentSystem
         public struct DisconnectData
         {
             public readonly int Length;
+            public EntityArray Entities;
             [ReadOnly] public SharedComponentDataArray<OnDisconnected> DisconnectMessage;
-
-            [ReadOnly] public ComponentDataArray<WorkerEntityTag> DenotesWorkerEntity;
+            [ReadOnly] public SharedComponentDataArray<WorkerConfig> Configs;
         }
 
         [Inject] private DisconnectData data;
 
         protected override void OnUpdate()
         {
-            Debug.LogWarningFormat("Diconnected from SpatialOS with reason: \"{0}\"",
-                data.DisconnectMessage[0].ReasonForDisconnect);
+            data.Configs[0].Worker.LogDispatcher.HandleLog(
+                LogType.Warning,
```

Comment:
Does this need to be a warning? Disconnecting is not abnormal flow

Reply:
I guess it depends on why you got disconnected. I simply converted it from `LogWarning` to our dispatcher, but can change it to Log if preferred. I don't have a strong opinion about this